examples/Classifier_Comparison.ipynb

```diff
@@ -43,7 +43,7 @@
     "using PyCall, Random, Statistics\n",
     "using PyPlot, Printf\n",
     "using ScikitLearn.CrossValidation: train_test_split\n",
-    "@pyimport matplotlib.colors as mplc\n",
+    "mplc = PyCall.pyimport_conda(\"matplotlib.colors\", \"matplotlib=3.2.2\")\n",
```

Comment:
This is still installing matplotlib 3.3.4 :thinking:

Reply:
Maybe it's installed by another `pyimport` encountered before this one?

Reply:
Ah, but maybe PyPlot installs matplotlib during `Pkg.add("PyPlot")`?